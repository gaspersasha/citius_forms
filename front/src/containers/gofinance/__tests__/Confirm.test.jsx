import React from 'react';
import { Confirm } from '..';

const props = {
  handleChange: jest.fn(),
  handleSubmit: jest.fn(),
  toggleModal: jest.fn(),
  isLoading: false,
  isDataSending: false,
  isLeadgen: false,
  handlePromoCodeChange: jest.fn(),
};

describe('<Confirm />', () => {
  it('orders snapshot', () => {
    const wrapper = mount(<Confirm {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
