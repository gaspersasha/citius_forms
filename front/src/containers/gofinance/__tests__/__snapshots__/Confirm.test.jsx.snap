// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Confirm /> orders snapshot 1`] = `
<Confirm
  handleChange={[MockFunction]}
  handlePromoCodeChange={[MockFunction]}
  handleSubmit={[MockFunction]}
  isDataSending={false}
  isLeadgen={false}
  isLoading={false}
  toggleModal={[MockFunction]}
>
  <div
    className="formFinance"
  >
    <div
      className="confirmationBlock"
    >
      <h3
        className="header"
      >
        Almost done!
      </h3>
      <img
        alt="almost_done"
        src="/images/almost_done_image.svg"
      />
    </div>
    <div
      className="lender"
    >
      <form
        className="formConfirm"
        onSubmit={[Function]}
      >
        <div
          className="optionOne"
        >
          <input
            className="confirmCheckbox"
            data-id="Find me my best available rate"
            id="send-to-lender"
            name="consent"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            htmlFor="send-to-lender"
          >
            <p>
              <b>
                Find me my best available rate
              </b>
            </p>
            <p>
              I consent to my application being sent to the lender and understand that this may result in one or more credit searches being registered
            </p>
          </label>
        </div>
        <Button
          className="confirm confirm_margin"
          disabled={false}
          href=""
          onClick={null}
          selected={false}
          styleType="primary"
          type="submit"
        >
          <button
            className="buttonComponent primary confirm confirm_margin"
            disabled={false}
            href={null}
            onClick={null}
            type="submit"
          >
            Submit my application
          </button>
        </Button>
      </form>
      <p
        className="submitting"
      >
        By Submitting your application, you are agreeing with ourÂ 
        <a
          href="#"
          onClick={[MockFunction]}
          role="button"
        >
          Application Terms
        </a>
      </p>
    </div>
  </div>
</Confirm>
`;
